sky/shell/platform/android/io/flutter/view/FlutterMain.java

```diff
@@ -277,22 +331,69 @@ public Binding connectToService(FlutterView view, Core core, MessagePipeHandle p
         });
     }
 
+    /**
+     * Initialize our Flutter config values by searching for a .properties file
+     * containing properties for our configuration variables. If no such
+     * properties file is found in our assets folder, the configuration
+     * variables will retain their default values.
```

Comment:
Searching around, it seems as though the poor performance of `AssetManager#list(String)` is well-known and that caching is a common strategy (as well as building an index file, which isn't feasible for frameworks).  So I'm gonna go with that, which should mean the overhead of this PR will be ~0